package.json

```diff
@@ -29,6 +29,6 @@
     "eslint-plugin-meteor": "^4.0.1"
   },
   "dependencies": {
-    "@okgrow/auto-analytics": "1.0.6"
+    "@okgrow/auto-analytics": "^2.0.0"
```

Comment:
We may have to keep this locked to `2.0.0` so its more clear and we don't end up with a conflict with `package-lock.json` being at a higher version then the `Npm.depends()` in Meteors package.json file.